### ISSUE TITLE
Move to Facebook C# SDK Organization

### ISSUE THREAD

**Issue by ntotten** [prabirshrestha/FluentHttp#3]
We should move this to the Facebook C# SDK organization.

**Comment by prabirshrestha** [comment]
I will need to do some code cleanup before shifting it there.

currently fb c# sdk uses only HttpHelper.cs file which is in dev branch at https://github.com/prabirshrestha/FluentHttp/blob/dev/src/FluentHttp/HttpHelper.cs

**Comment by prabirshrestha** [comment]
Should we move only the HttpHelper.cs into Facebook C# SDK organization as HttpHelper project?

It might even been good to rename the FluentHttp.Core nuget package to HttpHelper.

Currently HttpHelper and FluentHttp is a bit out of sync too. Now days I just prefer to use HttpHelper instead of FluentHttp.dll

**Comment by ntotten** [comment]
Yeah, that seems good to me. We could structure it and release it like simplejson, just in source not as a dll.
